assignment-sweet-compose/answer/docker-compose.yml

```diff
@@ -6,13 +6,12 @@
 version: '2.4'
 
 services:
-
   traefik:
```

Comment:
In the video it was called `proxy`, but the host is named `traefik.localhost`

Reply:
OK, I can change the service name to proxy to better match the video. Thanks.